src/context/GameContext.js

```diff
@@ -0,0 +1,116 @@
+import { createContext, useState } from 'react';
+
+const GameContext = createContext();
+
+const GameProvider = ({ children }) => {
+  const [currentPlayer, setCurrentPlayer] = useState('');
+  const [active, setActive] = useState(true);
+  const [gameMessage, setGameMessage] = useState(`It's ${currentPlayer}'s turn!`);
+  const [boardState, setBoardState] = useState([
+    { space: 1, content: '' },
+    { space: 2, content: '' },
+    { space: 3, content: '' },
+    { space: 4, content: '' },
+    { space: 5, content: '' },
+    { space: 6, content: '' },
+    { space: 7, content: '' },
+    { space: 8, content: '' },
+    { space: 9, content: '' },
+  ]);
+
+  function gameConditionCheck() {
+    //Increments for every box filled
+    let filledBoxes = 0;
+    boardState.map((box) => {
+      if (box.content !== '') {
+        filledBoxes++;
+      }
+    });
+
+    if (
+      //First Row Win
```

Comment:
You can simplify this a lot with something like (boardState[0].content === boardState[1].content && boardtate[1].content === boardstate[2]) 

Doing writing it in this fashion will eliminate you having to write for both X and O.